examples/datasets/plot_nutrimouse.py

```diff
@@ -0,0 +1,171 @@
+import seaborn as sns
+from mvlearn.datasets import load_nutrimouse
+
+dataset = load_nutrimouse()
+Xs = [dataset['gene'], dataset['lipid']]
+y = np.vstack((dataset['genotype'], dataset['diet'])).T
+
+print(f"Shapes of each view: {[X.shape for X in Xs]}")
+
+###############################################################################
+# Embed using MVMDS
+# -----------------
+#
+# Multiview multi-dimensional scaling embeds multiview data into a single
+# representation that captures information shared between both views.
+# Embedding the two nutrimouse views, we can observe clear separation between
+# the different genotypes and some of the diets too.
+
+from mvlearn.embed import MVMDS  # noqa: E402
+
+X_mvmds = MVMDS(n_components=2, num_iter=50).fit_transform(Xs)
+
+diet_names = dataset['diet_names']
+genotype_names = dataset['genotype_names']
+plt.figure(figsize=(5, 5))
+for genotype_idx, cmap in enumerate((cm.Blues, cm.Reds)):
+    for diet_idx in range(5):
+        X_idx = np.where((y == (genotype_idx, diet_idx)).all(axis=1))
+        color = cmap((diet_idx + 1) / 6)
+        label = diet_names[diet_idx] + f' ({genotype_names[genotype_idx]})'
+        plt.scatter(*zip(*X_mvmds[X_idx]), color=color, label=label)
+
+plt.xlabel('MVMDS component 1')
+plt.ylabel('MVMDS component 2')
+plt.title('MVMDS Embedding')
+plt.legend()
+plt.show()
+
+###############################################################################
+# Cluster using Multiview KMeans
+# ------------------------------
+#
+# We can compare the estimated clusters from multiview KMeans to regular
+# KMeans on each of the views. Multiview Kmeans clearly finds two clusters
+# matching the two different genotype labels observed in the prior plots.
+
+from mvlearn.cluster import MultiviewKMeans  # noqa: E402
+from sklearn.cluster import KMeans  # noqa: E402
+
+Xs_labels = MultiviewKMeans(n_clusters=2, random_state=0).fit_predict(Xs)
+X1_labels = KMeans(n_clusters=2, random_state=0).fit_predict(Xs[0])
+X2_labels = KMeans(n_clusters=2, random_state=0).fit_predict(Xs[1])
+
+sca_kwargs = {'alpha': 0.7, 's': 10}
+
+f, axes = plt.subplots(1, 3, figsize=(10, 3))
+axes[0].scatter(*zip(*X_mvmds), c=Xs_labels, **sca_kwargs)
+axes[0].set_title('Multiview Kmeans Clusters')
+axes[1].scatter(*zip(*X_mvmds), c=X1_labels, **sca_kwargs)
+axes[1].set_title('View 1 Kmeans Clusters')
+axes[2].scatter(*zip(*X_mvmds), c=X2_labels, **sca_kwargs)
+axes[2].set_title('View 2 Kmeans Clusters')
+
+for ax in axes:
+    ax.set_xlabel('MVMDS component 1')
+    ax.set_xticks([])
+    ax.set_yticks([])
+axes[0].set_ylabel('MVMDS component 2')
+axes[0].set_title('Multiview Kmeans Clusters')
+plt.show()
+
+###############################################################################
+# Decomposition using AJIVE
+# -------------------------
+#
+# We can also apply joint decomposition tools to find features across views
+# that are jointly related. Using AJIVE, we can find genes and lipids that are
```

Comment:
```suggestion
# that are jointly related. Using :class:`mvlearn.AJIVE`, we can find genes and lipids that are
```
something like this should work. It allows to have cross-links and point to the classes and more
details as AJIVE is certainly mysterious for most readers.